File: v2/jaxrs/src/main/java/io/swagger/model/Members.java
package io.swagger.model;

import java.util.*;
import io.swagger.model.Member;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Members  {
  
  private List<Member> posts = new ArrayList<Member>() ;
  private Boolean moreResults = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("posts")
  public List<Member> getPosts() {
    return posts;
  }
  public void setPosts(List<Member> posts) {
    this.posts = posts;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("moreResults")
  public Boolean getMoreResults() {
    return moreResults;
  }
  public void setMoreResults(Boolean moreResults) {
    this.moreResults = moreResults;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Members {\n");
    
    sb.append("  posts: ").append(posts).append("\n");
    sb.append("  moreResults: ").append(moreResults).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/io/swagger/Model/Members.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Members {
    

    
    public List<Member> Posts { get; set; }

    

    
    public bool? MoreResults { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Members {\n");
      
      sb.Append("  Posts: ").Append(Posts).Append("\n");
      
      sb.Append("  MoreResults: ").Append(MoreResults).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}